lib/migration/config/v1/executor/container_vm.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
I think that should be only applicable for update files. As well as we could use range 2016-2017.